process/project_admins.md

```diff
@@ -0,0 +1,27 @@
+# Project Admins
+
+A project admin is a TSC committer that has been granted GitHub administrative
```

Comment:
Instead of TSC committer, this should be changed to project committer. If the designated person(s) really need to be TSC members I would say TSC member and not committer as TSC members don't have to be committers of a given project (in general) I know that with ASWF it's a little more.... loose in that fashion in that each since at present each project basically has its own TSC, but that's not the general case for most projects we support.

Reply:
+1, on OpenCue there's not a lot of overlap between our TSC and our committers.

Reply:
Changed, I've gone for "project committer" as suggested.